lua/wikis/commons/ParticipantTable/Base.lua

```diff
@@ -172,10 +172,10 @@ function ParticipantTable:readSection(args)
 	local tbds = {}
 	Table.mapArgumentsByPrefix(args, {'p', 'player'}, function(key, index)
 		local entry = self:readEntry(args, key, index, config)
-		entry.sortName = Opponent.toName(entry.opponent)
+		entry.sortName = Opponent.toName(entry.opponent) or entry.opponent.template
```

Comment:
The fallback is just to support "delaying" missing team template error messages, correct?

Reply:
tbh basically copied it from the pr martin had
i think it is supposed to suppress errors due to missing/bad team templates and just showing the missing team template message in the display instead